.github/workflows/release.yml

```diff
@@ -11,6 +11,18 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Bazel caches
+        uses: actions/cache@v2
+        with:
+          # Only restore the repository cache. Don't restore the build cache
+          # as stamped bazel builds won't rebuild targets when the workplace
+          # status variables change.
+          path: |
+            ~/.cache/bazel-repo
+          key:
+            bazel-cache-${{ hashFiles('**/*.go', '**/BUILD.bazel', '**/*.bzl',
+            'WORKSPACE') }}
```

Comment:
Prettier wanted to put this on a separate line. I hope that doesn't break the key..

Reply:
It's subtle with YAML. Since you used `>` to start the string, it will replace the new lines `\n` with spaces.